planetiler-custommap/src/main/java/com/onthegomap/planetiler/custommap/ConfiguredMapMain.java

```diff
@@ -0,0 +1,79 @@
+package com.onthegomap.planetiler.custommap;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.onthegomap.planetiler.Planetiler;
+import com.onthegomap.planetiler.config.Arguments;
+import com.onthegomap.planetiler.custommap.configschema.DataSource;
+import com.onthegomap.planetiler.custommap.configschema.DataSourceType;
+import com.onthegomap.planetiler.custommap.configschema.SchemaConfig;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import org.yaml.snakeyaml.Yaml;
+
+public class ConfiguredMapMain {
```

Comment:
A little bit of javadoc on these classes in custommap package would help people find their way around.  It's not an API people will write code against, but just something to explain that this is the main driver that parses the yaml file into a profile and invokes PlanetilerRunner or something etc.